src/views/listview/list-view.html

```diff
@@ -1,52 +1,64 @@
-<div class="list-group list-view-pf">
+<div class="list-group list-view-pf" dnd-list="items" ng-class="{'list-view-pf-dnd': config.dragEnabled === true}">
+  <div class='dndPlaceholder'></div>
```

Comment:
I tried this and I don't quite understand why it behaves this way but you are correct. I don't see why the ng-if makes a difference but I guess we leave it as is.